lib/addons/fluxcd/index.ts

```diff
@@ -60,53 +111,65 @@ export interface FluxKustomizationProps {
  */
 export interface FluxCDAddOnProps extends HelmAddOnUserProps {
 
-  /**
-   * Namespace where add-on will be deployed. 
-   * @default flux-system
-   */
-  namespace?: string;
-
-  /**
-  * Helm chart version to use to install.
-  * @default 2.12.4
-  */
-  version?: string;
-
-  /**
-   * Values to pass to the chart as per https://github.com/argoproj/argo-helm/blob/master/charts/argo-cd/values.yaml.
-   */
-  values?: spi.Values;
-
-  /**
-   * To Create Namespace using CDK
-   */    
-  createNamespace?: boolean;
-
-  /**
-   * List of repositories to sync from.
-   */
-  repositories?: FluxGitRepo[];
+    /**
+     * Namespace where add-on will be deployed. 
+     * @default flux-system
+     */
+    namespace?: string;
+
+    /**
+     * Helm chart version to use to install.
+     * @default 2.13.0
+     */
+    version?: string;
+
+    /**
+     * Values to pass to the chart as per https://github.com/argoproj/argo-helm/blob/master/charts/argo-cd/values.yaml.
+     */
+    values?: spi.Values;
+
+    /**
+     * To Create Namespace using CDK
+     */    
+    createNamespace?: boolean;
+
+    /**
+     * List of repositories to sync from.
+     */
+    repositories?: FluxGitRepo[];
+
+    /**
+     * List of buckets to sync from.
+     */
+    buckets?: FluxBucketRepo[];
 }
 
 /**
  * Default props to be used when creating the Helm chart
  */
 const defaultProps: HelmAddOnProps & FluxCDAddOnProps = {
-  name: "fluxcd-addon",
-  namespace: "flux-system",
-  chart: "flux2",
-  version: "2.12.4",
-  release: "blueprints-fluxcd-addon",
-  repository: "https://fluxcd-community.github.io/helm-charts",
-  values: {},
-  createNamespace: true,
+    name: "fluxcd-addon",
+    namespace: "flux-system",
+    chart: "flux2",
+    version: "2.13.0",
+    release: "blueprints-fluxcd-addon",
+    repository: "oci://ghcr.io/fluxcd-community/charts/flux2",

```

Comment:
with the switch to OCR, are you aware of a mechanism for us to discover when new versions of flux become available (other than manual look up)?

Reply:
seems that `gh release` will show it but the output is not strictly for this chart:
```
$ gh release  list --repo fluxcd-community/helm-charts
flux2-sync-1.9.0           Latest  (flux2-sync-1.9.0)           about 1 month ago
flux2-notification-1.15.0          (flux2-notification-1.15.0)  about 1 month ago
flux2-2.13.0                       (flux2-2.13.0)               about 1 month ago
flux2-2.12.4                       (flux2-2.12.4)               about 3 months ago
flux2-sync-1.8.2                   (flux2-sync-1.8.2)           about 3 months ago
flux2-notification-1.14.1          (flux2-notification-1.14.1)  about 3 months ago
flux2-2.12.3                       (flux2-2.12.3)               about 3 months ago
flux2-notification-1.14.0          (flux2-notification-1.14.0)  about 4 months ago
flux2-sync-1.8.1                   (flux2-sync-1.8.1)           about 5 months ago
flux2-notification-1.13.1          (flux2-notification-1.13.1)  about 5 months ago
flux2-2.12.2                       (flux2-2.12.2)               about 5 months ago
flux2-2.12.1                       (flux2-2.12.1)               about 6 months ago
flux2-sync-1.8.0                   (flux2-sync-1.8.0)           about 6 months ago
flux2-notification-1.13.0          (flux2-notification-1.13.0)  about 6 months ago
flux2-2.12.0                       (flux2-2.12.0)               about 6 months ago
flux2-2.11.1                       (flux2-2.11.1)               about 7 months ago
flux2-sync-1.7.3                   (flux2-sync-1.7.3)           about 7 months ago
flux2-notification-1.12.4          (flux2-notification-1.12.4)  about 7 months ago
flux2-multi-tenancy-0.0.6          (flux2-multi-tenancy-0.0.6)  about 7 months ago
flux2-2.11.0                       (flux2-2.11.0)               about 7 months ago
flux2-2.10.6                       (flux2-2.10.6)               about 7 months ago
flux2-sync-1.7.2                   (flux2-sync-1.7.2)           about 7 months ago
flux2-notification-1.12.3          (flux2-notification-1.12.3)  about 7 months ago
flux2-2.10.5                       (flux2-2.10.5)               about 7 months ago
flux2-notification-1.12.2          (flux2-notification-1.12.2)  about 7 months ago
flux2-multi-tenancy-0.0.5          (flux2-multi-tenancy-0.0.5)  about 7 months ago
flux2-2.10.4                       (flux2-2.10.4)               about 7 months ago
flux2-2.10.3                       (flux2-2.10.3)               about 7 months ago
flux2-2.10.2                       (flux2-2.10.2)               about 8 months ago
flux2-sync-1.7.1                   (flux2-sync-1.7.1)           about 8 months ago
```
commented in 455a6a3fdf3a1bde30d42d232d306eb5a1bf2992